packages/shared/locales/en.json

```diff
@@ -1046,6 +1046,7 @@
         "received": "Received",
         "transfer": "Transfer",
         "transferring": "Transferring",
+        "shimmerStaking": "Shimmer Staking",
```

Comment:
This might potentially be confusing when we bring a proper staking mechanism to the Shimmer network. Perhaps `Shimmer Claim` or `Claimed Shimmer` might be better?

Reply:
Will we bring staking? Can always change it then. Perhaps something like from "Shimmer genesis"?

Reply:
Lets go with:
**Claimed**
From Shimmer Genesis

Reply:
> Will we bring staking?

Honestly not sure, but I was under the impression that we were eventually. Not sure though after the announcement just a few minutes ago... :sweat_smile:

@MarkNerdi We decided on this design for the tile asset (see Discord):

**Claimed**
from Shimmer Genesis